test/addons-napi/test_promise/test.js

```diff
@@ -6,6 +6,8 @@ const assert = require('assert');
 
 let expected_result, promise;
 
+process.on('unhandledRejection', () => {});
```

Comment:
Can we fix the test instead of ignoring the errors?

If we can't it might indicate a design problem

Reply:
@addaleax recommended to ignore this. I did not find the cause of the issue here especially as the output is really little.